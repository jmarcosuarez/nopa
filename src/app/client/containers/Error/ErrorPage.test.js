/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import ErrorPage from './ErrorPage';

describe('Renders correctly', () => {
  it('should render correctly', () => {
    const output = shallow(
      <ErrorPage route={{ error: '404' }} />,
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
