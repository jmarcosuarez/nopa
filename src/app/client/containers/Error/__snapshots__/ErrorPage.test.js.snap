// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly should render correctly 1`] = `
<Layout
  title=""
>
  <div>
    <h1>
      Page not found!
    </h1>
    <div>
      This page is either no longer available or you have entered a wrong path.
    </div>
  </div>
</Layout>
`;
